src/containerbuddy/config.go

```diff
@@ -102,6 +142,24 @@ func loadConfig() (*Config, error) {
 		return nil, err
 	}
 
+	onStartCmd, err := parseCommandArgs(config.OnStart)
+	if err != nil {
+		return nil, errors.New(fmt.Sprintf("Could not parse `onStart`: %s", err.Error()))
```

Comment:
Just from the standpoint of verbosity: you don't need to call the `.Error()` method on these. Errors just wrap strings in golang so they have the interfaces to be `%s`-formatted. In fact, as I just discovered when looking up something to give you as a reference, this particular construct is supported in the stdlib (ref https://golang.org/pkg/errors/#example_New_errorf). So try:

```go
if err != nil {
    return nil, fmt.Errorf("Could not parse `onStart`: %s", err)
}
```